web/package.json

```diff
@@ -29,6 +29,7 @@
     "react-select": "^2.4.2",
     "react-test-renderer": "^16.7.0",
     "react-timeago": "^4.4.0",
+    "styled-components": "^4.2.0",
```

Comment:
did you actually use this anywhere? i couldn't find it...

Reply:
Thanks for catching this! Must have been a dep I missed out when I uninstalled the spinner kit that DM added.